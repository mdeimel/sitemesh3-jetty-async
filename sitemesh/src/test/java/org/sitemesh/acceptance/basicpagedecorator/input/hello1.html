<html>
  <head>
    <title>Hello 1</title>
  </head>
  <body>

   <p>Very simple document.</p>

  </body>
</html>