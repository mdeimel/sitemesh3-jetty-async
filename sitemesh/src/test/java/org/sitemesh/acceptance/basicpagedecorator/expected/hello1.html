<!-- Common header -->
<html>
  <head>
    <title>Hello 1</title>
    <style type="text/css">
      /* common css */
      body { padding: 20px; }
    </style>
  </head>
  <body>

    <h1 style="text-align:center">Hello 1</h1>

    <!-- body -->
    <p>Very simple document.</p>
    <!-- end of body -->

  </body>
</html>
<!-- Common footer -->